src/EPR.RegulatorService.Frontend.Web/Controllers/Applications/ApplicationsController.cs

```diff
@@ -462,7 +473,7 @@ public async Task<IActionResult> AcceptApplication(string? acceptedUserFirstName
             }
             catch (Exception e)
             {
-                _logger.LogError(e, $"Error accepting application for: {acceptedUserEmail} in organisation {organisationDetails.OrganisationId}");
+                _logErrorAcceptingApplication(_logger, acceptedUserEmail, organisationDetails.OrganisationId, e);
```

Comment:
same here, this does not look right

Reply:
@hardik-desai-kcl  SQ complaining as on this line we need use LoggerMessage.

_logger.LogError(e, $"Error accepting application for: {acceptedUserEmail} in organisation {organisationDetails.OrganisationId}");

For improved performance, use the LoggerMessage delegates instead of calling 'LoggerExtensions.LogError(ILogger, string?, params object?[])'

Updated the code to fix nothing major change it's just way of defining loggers.

Reply:
try this
_logger.LogError(e, "Error accepting application for: {AcceptedUserEmail} in organisation {OrganisationId}", acceptedUserEmail, organisationDetails.OrganisationId);